models/contracts/optimism/contracts_optimism_contract_mapping.sql

```diff
@@ -159,6 +159,35 @@ with base_level as (
   left join {{ source('optimism', 'contracts') }} as oc 
     on cc.contract_address = oc.address 
 
+  union all
+  -- missing contracts
+  select 
+     COALESCE(oc.`from`,'0xdeaddeaddeaddeaddeaddeaddeaddeaddead0006') AS trace_creator_address
+    ,COALESCE(oc.`from`,'0xdeaddeaddeaddeaddeaddeaddeaddeaddead0006') AS creator_address
```

Comment:
Maybe we can remove \`
```suggestion
     COALESCE(oc.from,'0xdeaddeaddeaddeaddeaddeaddeaddeaddead0006') AS trace_creator_address
    ,COALESCE(oc.from,'0xdeaddeaddeaddeaddeaddeaddeaddeaddead0006') AS creator_address
```